app/Models/Litter/Categories/Brand.php

```diff
@@ -106,110 +107,88 @@ class Brand extends LitterCategory
-            'wrigleys'
+            'adidas' => 'adidas',
+            'albertheijn' => 'albertheijn',
+            'amazon' => 'amazon',
+            'aldi' => 'aldi',
+            'apple' => 'apple',
+            'applegreen' => 'applegreen',
+            'asahi' => 'asahi',
+            'avoca' => 'avoca',
+            'ballygowan' => 'ballygowan',
+            'bewleys' => 'bewleys',
+            'brambles' => 'brambles',
+            'budweiser' => 'budweiser',
+            'bulmers' => 'bulmers',
+            'burgerking' => 'burgerking',
+            'butlers' => 'butlers',
+            'cadburys' => 'cadburys',
+            'cafe_nero' => 'cafe_nero',
+            'camel' => 'camel',
+            'carlsberg' => 'carlsberg',
+            'centra' => 'centra',
+            'coke' => 'coca_cola',
+            'circlek' => 'circlek',
+            'coles' => 'coles',
+            'colgate' => 'colgate',
+            'corona' => 'corona',
+            'costa' => 'costa',
+            'doritos' => 'doritos',
+            'drpepper' => 'drpepper',
+            'dunnes' => 'dunnes',
+            'duracell' => 'duracell',
+            'durex' => 'durex',
+            'evian' => 'evian',
+            'esquires' => 'esquires',
+            'fosters' => 'fosters',
+            'frank_and_honest' => 'frank_and_honest',
+            'fritolay' => 'fritolay',
+            'gatorade' => 'gatorade',
+            'gillette' => 'gillette',
+            'guinness' => 'guinness',
+            'haribo' => 'haribo',
+            'heineken' => 'heineken',
+            'insomnia' => 'insomnia',
+            'kellogs' => 'kellogs',
+            'kfc' => 'kfc',
+            'lego' => 'lego',
+            'lidl' => 'lidl',
+            'lindenvillage' => 'lindenvillage',
+            'lolly_and_cookes' => 'lolly_and_cookes',
+            'loreal' => 'loreal',
+            'lucozade' => 'lucozade',
+            'marlboro' => 'marlboro',
+            'mars' => 'mars',
+            'mcdonalds' => 'mcdonalds',
+            'nero' => 'nero',
+            'nescafe' => 'nescafe',
+            'nestle' => 'nestle',
+            'nike' => 'nike',
+            'obriens' => 'obriens',
+            'pepsi' => 'pepsi',
+            'powerade' => 'powerade',
+            'redbull' => 'redbull',
+            'ribena' => 'ribena',
+            'samsung' => 'samsung',
+            'sainsburys' => 'sainsburys',
+            'spar' => 'spar',
+            'stella' => 'stella',
+            'subway' => 'subway',
+            'supermacs' => 'supermacs',
+            'supervalu' => 'supervalu',
+            'starbucks' => 'starbucks',
+            'tayto' => 'tayto',
+            'tesco' => 'tesco',
+            'thins' => 'thins',
+            'volvic' => 'volvic',
+            'waitrose' => 'waitrose',
+            'walkers' => 'walkers',
+            'woolworths' => 'woolworths',
+            'wilde_and_greene' => 'wilde_and_greene',
+            'wrigleys' => 'wrigleys',
```

Comment:
seems like a lot of duplication

Reply:
It's the only way to automatically test that the columns on the CSV exports are the same as the tags. It's a necessary duplication for now to assert the data correctness. Of course, I intend to refactor this even further by introducing translations and most likely go back to the single types() array, but this is a necessary step in that direction.

And by the way, we still have and use the types() array on every category, it's declaration is simply refactored up to their parent LitterCategory. And it returns the same identical data as before.